cirq/experiments/t2_decay_experiment.py

```diff
@@ -68,10 +69,36 @@ def t2_decay(
     However, during the mid-point of the delay time being measured, a pi-pulse
     (`cirq.X`) gate will be applied to cancel out inhomogeneous dephasing.
     The same method of measuring the final state as Ramsey experiment is applied
-    after the second half of the delay period.
-
-    CPMG, or the Carr-Purcell-Meiboom-Gill sequence, is currently not
-    implemented.
+    after the second half of the delay period.  See the animation on the wiki
+    page https://en.wikipedia.org/wiki/Spin_echo for a visual illustration
+    of this experiment.
+
+    CPMG, or the Carr-Purcell-Meiboom-Gill sequence, involves using a sqrt(Y)
+    followed by a sequence of pi pulses (X gates) in a specific timing pattern:
+        π/2, t, π, 2t, π, ... 2t, π, t
```

Comment:
nit: maybe add empty line before and after for readability and consistency with other docstrings in cirq.

Reply:
Done.